### PR TITLE
Devx 606

### PR DESCRIPTION
**Pull request by reekamaharaj**
Links to reach-lang repo will open in a new tab, links on the navbar will open in a new tab.

### CODE REVIEW

File: docs/gen/generator.js [hunk 1]
```diff
@@ -222,7 +222,9 @@ const processXRefs = ({here}) => (tree) => {
       const u = node.url;
       if ( u && u.startsWith("##") ) {
         node.url = xrefGet('h', u.slice(2)).path;
-      }
+      } else if (u.startsWith('https')) {
+		hp.target = `_blank`;
```
Comment (TheChronicMonster) [inline docs/gen/generator.js:226]: This line looks like it's indented further than needed.

File: docs/gen/generator.js [hunk 2]
```diff
@@ -222,7 +222,9 @@ const processXRefs = ({here}) => (tree) => {
       const u = node.url;
       if ( u && u.startsWith("##") ) {
         node.url = xrefGet('h', u.slice(2)).path;
-      }
+      } else if (u.startsWith('https')) {
```
Comment (TheChronicMonster) [inline docs/gen/generator.js:225]: Add spaces inside the parens to match the styling on line 223.

**Review by TheChronicMonster — CHANGES_REQUESTED** [review]

**Review by TheChronicMonster — APPROVED** [review]

File: docs/gen/generator.js [hunk 3]
```diff
@@ -222,7 +222,9 @@ const processXRefs = ({here}) => (tree) => {
       const u = node.url;
       if ( u && u.startsWith("##") ) {
         node.url = xrefGet('h', u.slice(2)).path;
-      }
+      } else if ( u.startsWith('https') ) {
+        hp.target = `_blank`;
+	    }
```
Comment (jeapostrophe) [inline docs/gen/generator.js:227]:
```suggestion
      } else if ( u.startsWith('https') ) {
        hp.target = `_blank`;
      }
```

Comment (jeapostrophe) [reply docs/gen/generator.js]: No heinous demonic tabs

**Review by jeapostrophe — APPROVED** [review]

**Pull request merged**